scania-digital-design-system/tegel
# feat(icons): add new document icons

**Describe pull-request**  
Added new PDF, PPT, XLS and DOC icons to Storybook. Icons have also been published to our webfont CDN solution and are prepared for our next website release.

**Solving issue**  
Fixes: [DTS-1336](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?selectedIssue=DTS-1336) and  [DTS-1483](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?selectedIssue=DTS-1483)

**How to test**  
1. Go to Storybook link below
2. Check in Foundations -> Icons -> Native and Web Components
3. In the list of icons, choose "document_doc", "document_pdf", "document_ppt" and "document_xls"
4. Make sure the right icon is displayed

[DTS-1336]: https://tegel.atlassian.net/browse/DTS-1336?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DTS-1483]: https://tegel.atlassian.net/browse/DTS-1483?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
